package.json

```diff
@@ -114,29 +114,29 @@
     "css-loader": "^0.28.4",
     "cz-conventional-changelog-components": "^1.0.0",
     "enzyme": "^3.1.0",
-    "enzyme-adapter-react-15": "^1.0.1",
+    "enzyme-adapter-react-16": "^1.1.0",
     "enzyme-to-json": "^3.1.2",
-    "eslint": "^3.0.0",
+    "eslint": "^4.11.0",
     "eslint-plugin-jsx-a11y": "^2.0.0",
-    "eslint-plugin-react": "^6.0.0",
-    "gh-pages": "0.11.0",
-    "husky": "^0.13.1",
-    "jest": "^20.0.4",
+    "eslint-plugin-react": "^7.4.0",
+    "gh-pages": "1.0.0",
+    "husky": "^0.14.3",
+    "jest": "^21.2.1",
     "lcov2badge": "^0.1.0",
-    "lint-staged": "^3.4.0",
-    "node-sass": "3.8.0",
+    "lint-staged": "^4.3.0",
+    "node-sass": "4.6.0",
     "object-assign": "^4.1.1",
     "postcss-loader": "^2.0.5",
     "prettier": "^1.7.4",
     "promise": "^8.0.1",
     "prop-types": "^15.5.8",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4",
-    "react-test-renderer": "^15.6.1",
+    "react": "^16.1.0",
```

Comment:
Should we update our peer dependencies too?

Reply:
Thank you for catching! Done.